src/System.IO.Pipelines/ref/System.IO.Pipelines.cs

```diff
@@ -72,9 +74,14 @@ public abstract partial class PipeWriter : System.Buffers.IBufferWriter<byte>
     public readonly partial struct ReadResult
     {
         private readonly object _dummy;
+        private readonly int _dummyPrimitive;
```

Comment:
Why is there both a _dummy and _dummyPrimitive?

Reply:
See https://github.com/dotnet/corefx/issues/6185#issuecomment-185808210.

Reply:
1. This is auto-generated
2. The goal is to match this layout https://github.com/dotnet/corefx/blob/87de1b25d5c71283cc8c53d90fdace453d0916e7/src/System.IO.Pipelines/src/System/IO/Pipelines/ReadResult.cs#L14-L15.